src/errors.rs

```diff
@@ -15,8 +15,9 @@ pub enum Error {
     #[error(transparent)]
     MeilisearchCommunication(#[from] MeilisearchCommunicationError),
     /// The Meilisearch server returned an invalid JSON for a request.
-    #[error("Error parsing response JSON: {}", .0)]
-    ParseError(#[from] serde_json::Error),
+    #[error(transparent)]
+    SerdeParseError(#[from] serde_json::Error),
```

Comment:
```suggestion
    #[error("Error parsing response JSON: {}", .0)]
    ParseError(#[from] serde_json::Error),
```

Hey, I don't think we should rename this error name, it's a breaking change that doesn't brings much to the end user